src/VisualStudio/Core/Def/Implementation/FindReferences/StreamingFindUsagesPresenter.cs

```diff
@@ -58,7 +59,11 @@ internal partial class StreamingFindUsagesPresenter :
                    typeMap,
                    formatMapService,
                    classificationFormatMapService,
-                   columns.Where(c => c.Metadata.Name == FindUsagesValueUsageInfoColumnDefinition.ColumnName).Select(c => c.Value))
+                   columns.Where(c =>
```

Comment:
Do we have to rely on `ITableColumnDefinition` here? It seems we could instead have an intermediate interface that we use for MEF here instead of having to update a bunch of name comparisons. 

Alternatively, `c.Value is AbstractCustomColumnDefinition` instead of checking names

Reply:
+1 that would be more generic check

---
In reply to: [313623080](https://github.com/dotnet/roslyn/pull/37802#discussion_r313623080) [](ancestors = 313623080)

Reply:
This should still be implemented as a type check as @ryzngard suggested.

Reply:
@chborl Would you be able to fix this?